hunt/README.md

```diff
@@ -36,3 +36,8 @@ A few key gems to read up on for this project are the following:
 | ---------------------------------------------------------------------------- | :------------------------------------------------: |
 | [Bootstrap](https://getbootstrap.com/docs/4.5/getting-started/introduction/) | A CSS & Javascript Framework For Building Websites |
 | [SimpleForm Bootstrap](http://simple-form-bootstrap.plataformatec.com.br/)   |     Examples of using SimpleForm w/ Bootstrap      |
+
+## Running the Rake Task for Animal Import from Spreadsheet
+
+- Make sure you are in the hunt directory in the terminal.
+- Type "rake import:animals" to run the rake task.
```

Comment:
Instead of using quotations here, you can use backticks (``) to format this as code. Like this: `rake import:animals`

```
You can use three back ticks (```) on either side of the text for a code block
```